### PR TITLE
Add Calltouch datasource docs | SS-20

### PR DESCRIPTION
**Pull request by MaksimKiselev**

### CODE REVIEW

File: _docs/datasources/calltouch.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+After enabling this data source, Calltouch calls history will be uploaded to BigQuery once every 24 hours.
+
+### Connecting and configuring
+
+To enable API access in Calltouch dashboard, go to project, click "Интеграции" (1) > "API и Webhooks" (2)
+
+![](/img/calltouch_1.png)
+
+Ensure that radio button (1) is enabled. Copy Site ID (2) and token (3). If token not created yet, click "Обновить" button (4) to create it.
```
Comment (ArtemBogomolov) [inline _docs/datasources/calltouch.md]:
```suggestion
Ensure that the radio button (1) is enabled. Copy the Site ID (2) and token (3). If the token hasn't been created yet, click the "Обновить" button (4) to create it.
```

File: _docs/datasources/calltouch.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+> Attention! The [Google BigQuery](/integrations/google-bigquery) integration has to be enabled to use this feature.
+
+### Importing calls from Calltouch
+
+After enabling this data source, Calltouch calls history will be uploaded to BigQuery once every 24 hours.
+
+### Connecting and configuring
+
+To enable API access in Calltouch dashboard, go to project, click "Интеграции" (1) > "API и Webhooks" (2)
```
Comment (ArtemBogomolov) [inline _docs/datasources/calltouch.md]:
```suggestion
To enable API access to the Calltouch dashboard, go to your project, and click "Интеграции" (1) > "API и Webhooks" (2)
```

File: _docs/datasources/calltouch.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+![](/img/calltouch_1.png)
+
+Ensure that radio button (1) is enabled. Copy Site ID (2) and token (3). If token not created yet, click "Обновить" button (4) to create it.
+
+![](/img/calltouch_2.png)
+
+The process of connecting data sources is described in detail in the [overview](https://docs.segmentstream.com/datasources/index).
+
+For authorization put API token (1) from previous step and click "Save" (2)
```
Comment (ArtemBogomolov) [inline _docs/datasources/calltouch.md]:
```suggestion
For authorization insert the API token (1) from the previous step and click "Save" (2) 
```

File: _docs/datasources/calltouch.md (hunk 1 above)
Comment (MaksimKiselev) [reply _docs/datasources/calltouch.md]: Thx.

File: _docs/datasources/calltouch.md (hunk 2 above)
Comment (MaksimKiselev) [reply _docs/datasources/calltouch.md]: Thx.

File: _docs/datasources/calltouch.md (hunk 3 above)
Comment (MaksimKiselev) [reply _docs/datasources/calltouch.md]: Thx.

**Pull request merged**